modules/ca/variables.tf

```diff
@@ -20,4 +20,14 @@ variable "tags" {
   default = {
     terraform = true
   }
+}
+
+variable "s3_access_log_bucket" {
+  type        = string
+  description = "What bucket to write access logs to"
```

Comment:
dont forget about default values for this module. Also I think we can use `null` rather than `""` but that will depend on if the s3 bucket resource can take a null value or not.

Reply:
how'd I miss that ... 🤦